src/Repository/InvoiceRepository.php

```diff
@@ -4,6 +4,8 @@
 
 namespace Sylius\InvoicingPlugin\Repository;
 
+@trigger_error('The "InvoiceRepository" interface is deprecated since version 0.11.1 Use standardized interface located at "src/Doctrine/ORM/" instead.');
```

Comment:
As it is not a stable version of the plugin yet, I would be for removing deprecation and just refactor the approach to `InvoiceRepository` as you did in this PR.

Reply:
Done and I add a note the `UPGRADE.md`.